.github/ISSUE_TEMPLATE/redeliver_container_image.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
We're not delivering container images. Can we drop this issue template?

Reply:
yes, surely.